### DISCUSSION TITLE
palette changes based on backgrounds/entities

### DISCUSSION THREAD

**Discussion by city41** [city41/smaghetti#147] [General]
polterbryce has been documenting these in discord. Capturing them here to have a more permanent place for them

![image](https://user-images.githubusercontent.com/141159/137609442-b3f997c6-3254-4dde-8e6e-194f3dcd1d81.png)

![image](https://user-images.githubusercontent.com/141159/137609460-bdba9ddc-350b-4ba5-aea6-810d427e0ac8.png)

![image](https://user-images.githubusercontent.com/141159/137609456-2b5b8db5-718f-4126-959a-2bf14a1a66f9.png)

**Comment by lelegofrog** [comment]
palettes i found for stone block, not sure if a block overrides the palette
![image](https://user-images.githubusercontent.com/61850335/137610394-609b6f9e-20f5-4fb4-913c-aba261efca6c.png)
